hack/packages/package-tools/cmd/vendir-sync.go

```diff
@@ -43,40 +44,50 @@ func runPackageVendirSync(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	var g errgroup.Group
```

Comment:
I know we Go programmers try to keep variables as short as possible, but the caveat is that it needs to be clear what it means. I think g is used far enough from its declaration, it should be named something like errGrp to be clear.

Reply:
I can also just move this declaration to L55. Would that be clearer or would you still like it to be renamed? `var errGrp errgroup.Group` is too much stutter.

Reply:
L55 would be better. But what do you think about naming it `errs`?
Would `var errs errgroup.Group` be more confusing or more clear?

Reply:
After looking at docs, I think `g` is ok. Maybe it's common enough that it will not be confusing.
![Screen Shot 2022-07-22 at 9 07 11 AM](https://user-images.githubusercontent.com/17328443/180479716-603d44be-4544-4e5c-98c8-93d42e53106b.png)

Reply:
`errs` indicate a slice of errors, so it would be confusing. Typically, people use `g` or `wg` for error/waitgroups like this.